subsys/bluetooth/host/Kconfig

```diff
@@ -163,12 +163,15 @@ rsource "../mesh/Kconfig"
 rsource "../audio/Kconfig"
 
 config BT_HOST_CRYPTO
-	# Hidden option that compiles in AES encryption support using TinyCrypt
-	# library if this is not provided by the controller implementation.
-	bool
+	bool "Use crypto functionality implemented in the Bluetooth host"
 	default y if !BT_CTLR_CRYPTO
 	select TINYCRYPT
 	select TINYCRYPT_AES
+	help
+	  The option adds the AES encryption support using TinyCrypt
+	  library if this is not provided by the controller implementation.
+	  The option is not required if BLE Mesh is enabled and
```

Comment:
Not sure if mesh should be mentioned here. What about `bt_encrypt_le` and `bt_encrypt_be` functions? They aren't replaced when mesh is enabled with PSA support. This options still needs to be enabled if a customer wants to enable SMP (Security Manager Protocol, not SMP from MCUmgr).

Reply:
Mesh doesn't use them. Yes, if customer adds anything that uses them, tinycrypt will appear again. I guess there is no common strategy to get rid tinycrypt in Zephyr in general.